README.md

```diff
@@ -22,7 +22,7 @@ cloud (VPC). The [examples](examples/) directory contains complete automation sc
 The AWS Terraform scripts leverage Terraform v1.1.9 which includes full binary and provider support for macOS M1 chips, but any Terraform
 version 0.13.7 should be generally supported.
 
--   provider registry.terraform.io/hashicorp/aws v4.7.x
+-   provider registry.terraform.io/hashicorp/aws v4.46.x
```

Comment:
Currently, there is v4.48.  Not that the changes (b/w v.46 and v.48) are not necessary as those fixes/new enhancements are needed but good to be on the latest version while we are not GA yet.  Yes, v.4.46 changes has a direct impact on this PR and v4.38.0 (stickiness and failover)

Reply:
that's fair. .46 was the latest stable when I started writing this as all. hard to stay up to date with aws :)

I can update all the version to latest all of 12/20/22 though